README.md

```diff
@@ -272,6 +272,13 @@ This is useful when installing your application on your device using the Develop
 
 You can statically select the right provisioning profile in your Xcode project (the name will be `match Development tools.fastlane.app`).
 
+### Continuous Integration
+
+When using match with a CI you'll need to do two things:
+
+- Set your encryption password as secret environment variable named `MATCH_PASSWORD`
+- Remember to give your CI access to the repo containing your keys, certs and profiles. You'll likely have to make a "bot" account on your Git host since you'll already have used the CI's deploy key for codebase repo.
```

Comment:
> You'll likely have to make a "bot" account on your Git host since you'll already have used the CI's deploy key for codebase repo.
This doesn't seem to fit here, can we remove this sentence?